### ISSUE TITLE
How to deal with source mapping changes

### ISSUE THREAD

**Issue by sprice** [videojs/font#10]
The audio description icon used to be `\f114` and now it's `\f11d`. What's the best way to build an external plugin using an icon from this package without relying on these mappings?

**Comment by gkatsev** [comment]
Hm... well, hopefully we never have this change. Not really sure how things are generated but we should probably put the audio description icon back to \f114. Or At this point maybe even keep both...

**Comment by gkatsev** [comment]
Also, I think we probably should move away from using the codepoint altogether and use classes to target things so if the codepoints do change, we won't have issues.

**Comment by sprice** [comment]
`\f114` is now another icon.

I for one certainly like the idea of never changing the codepoints :smile: Classes are a great idea as well.

**Comment by mmcc** [comment]
I went down the path of trying to make sure codepoints get locked down, but that's a can of worms giving the tooling we have available :dizzy_face:. After thinking through it for a while, I don't think it would ever be a good idea for us to advertise that the codepoints will never change (even if we did lock them down), just because of how inflexible that would make things for us.

Anyway, everything is already targeted by classes. When the font is generated, it automatically generates the CSS / SCSS files with the correct mappings, then Video.js imports the styles (SCSS for VJS) into its own work flow. In a plugin that assumes the Video.js CSS exists, the icon classes will merrily stay updated for you all the time :)

[All the CSS classes](https://github.com/videojs/font/blob/master/css/videojs-icons.css)
